src/Standards/Generic/Docs/Functions/OpeningFunctionBraceKernighanRitchieStandard.xml

```diff
@@ -1,22 +1,23 @@
-<documentation title="Opening Brace in Function Declarations">
+<documentation title="Opening Function Brace Kerninghan Ritchie">
     <standard>
     <![CDATA[
-    Function declarations follow the "Kernighan/Ritchie style". The function brace is on the same line as the function declaration. One space is required between the closing parenthesis and the brace.
+    The function opening brace must be on the same line as the end of the function declaration, with exactly
+    one space between them, and must be the last content on the line.
```

Comment:
Oh shoot, I'm still seeing an issue - "one space between them" - what's the them ?

Reply:
I like the second option - what about the below (= second option with very minor tweak):

"The function opening brace must be on the same line as the end of the function declaration, with exactly one space between the end of the declaration and the brace. The brace must be the last content on the line."